### ISSUE TITLE
[Check] Request data into cursor.execute(...)

### ISSUE THREAD

**Issue by minusworld** [returntocorp/sgrep-rules#283]
**Description**
This is similar to, if not identical to, our other SQLi checks for extra and Raw SQL, but for `cursor.execute`.

This comes from a blog post highlighting the following pattern:
```python
def search(request):
    query = request.GET['q']
    sql = f"SELECT * FROM some_table WHERE title LIKE '%{query}%';"
    
    cursor = db.cursor()
    cursor.execute(sql)
    ...
```

- [x] This ticket has links, references, or examples.
- [x] Your check has true positive and true negative test cases.
- [x] Your check has been labeled with true positive and false positive findings on at least 10 repositories, and the link is pasted in the ticket.
- [x] Your check PR has been reviewed and merged.
Now, close the ticket!

**Comment by ulziibay** [comment]
https://github.com/returntocorp/sgrep/issues/326 would unblock this!

**Comment by ulziibay** [comment]
https://dev.massive.ret2.co/jobs/773?page=1&sort_on=repo_url&sort_by=asc
